qiskit_optimization/applications/tsp.py

```diff
@@ -157,20 +157,23 @@ def parse_tsplib_format(filename: str) -> "Tsp":
             for line in infile:
                 if line.startswith("NAME"):
                     name = line.split(":")[1]
-                    name.strip()
+                    name = name.strip()
                 elif line.startswith("TYPE"):
                     typ = line.split(":")[1]
-                    typ.strip()
+                    typ = typ.strip()
                     if typ != "TSP":
                         raise QiskitOptimizationError(
                             f'This supports only "TSP" type. Actual: {typ}'
                         )
+                elif line.startswith("EOF"):
+                    # End Of File tag
+                    continue
```

Comment:
I feel it would be better to `break` here from the loop, when hitting an EOF, rather than continuing to parse any further lines that may be there in the file.

Reply:
Agree, I pushed the edit and release note.